src/lib/ursadb.py

```diff
@@ -62,15 +75,13 @@ def pop(self, iterator: str, count: int) -> Tuple[bool, List[str]]:
 
         res = json.loads(response)
         if "error" in res:
```

Comment:
:thinking:

Reply:
We should probably document ursadb response scheme :thinking:

Reply:
this is prety fragile as it relies on the existence of the key rather than value; also with "statically typed" objects this should be either always false or always true

Reply:
See below:

```python
res["error"].get("retry", False)`
```
So we don't rely only on a key existence, we just treat missing key as False